unity/Holo/Assets/Editor/ModelLoad/ModelInfoClasses.cs

```diff
@@ -15,7 +15,7 @@ public class ModelInfo
     public class ModelLayerInfo
     {
 	    public string Caption;
-		public bool Simulation;
+		public string Simulation;

```

Comment:
I would like a more descriptive name, or at least a comment explaining what it is for

Reply:
We've talked about this, this will be unified in the next PR. :)